frontend/src/scenes/insights/views/LineGraph/LineGraph.tsx

```diff
@@ -644,12 +663,16 @@ export function LineGraph_({
                 },
                 y: {
                     display: !hideYAxis,
-                    beginAtZero: true,
+                    type: isLog10 ? 'logarithmic' : 'linear',
+                    beginAtZero: true, // Note that `beginAtZero` has no effect on the log scale
+                    // Below guarding against LOG_ZERO being the minimum in log scale, as that would make the graph
+                    // hard to read (due to the multiple orders of magnitude between `seriesNonZeroMin` and `LOG_ZERO`)
+                    min: isLog10 ? Math.pow(10, Math.ceil(Math.log10(seriesNonZeroMin)) - 1) : undefined,
                     stacked: showPercentStackView || isArea,
                     ticks: {
-                        display: !hideYAxis,
                         ...tickOptions,
-                        precision,
+                        display: !hideYAxis,
+                        ...(yAxisScaleType !== 'log10' && { precision }), // Precision is not supported for the log scale
```

Comment:
I can't leave precision as is here because types don't match then, because precision is not a field for log scale.

Reply:
Oh of course, I didn't notice TypeScript alerting me about this in VS Code, so missed that